telco-hub/configuration/reference-crs/optional/cert-manager/certManagerSubscription.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+  labels:
+    operators.coreos.com/openshift-cert-manager-operator.cert-manager-operator: ''
```

Comment:
I believe this is automatically added by OLM and can be left off

Reply:
Removed.